### PR TITLE
Jillian added a new line of text.

### PR DESCRIPTION
**Pull request by jmburns000**
Hi Ben, 

I added a new line of text. Please review and let me know if it works. 

Best,
Jillian

### CODE REVIEW

File: data-cleaning.rmd [hunk 1]
```diff
@@ -9,4 +9,6 @@ output: html_document
 knitr::opts_chunk$set(echo = TRUE)
 ```
 
-## Data Cleaning
\ No newline at end of file
+## Data Cleaning
+
+This is a new method to clean data. Jillian added the new method below.
```
Comment (bsgerber) [inline data-cleaning.rmd]: I'd rather this were a subsection.

Comment (jmburns000) [reply data-cleaning.rmd]: Ok, I understand.

Comment (jmburns000) [reply data-cleaning.rmd]: I will make the change.

Comment (bsgerber) [reply data-cleaning.rmd]: Ok - I'll await your update.

**Comment by jmburns000** [comment]
Hi Ben, 

Just made the change. Let me know you thoughts.

**Comment by bsgerber** [comment]
looks great and merged!

**Pull request merged**